docs/agent/secrets.md

```diff
@@ -247,3 +259,30 @@ instances:
     password: decrypted_db_prod_password
 ```
 
+### Troubleshooting
+
+To quickly see how the configurations are resolved you can use the `configcheck` command :
+
+```shell
+sudo -u dd-agent -- datadog-agent configcheck
+
+=== a check ===
+Source: File Configuration Provider
+Instance 1:
+host: <decrypted_host>
+port: <decrypted_port>
+passowrd: <decrypted_password>
+~
+===
+
+=== another check ===
+Source: File Configuration Provider
+Instance 1:
+host: <decrypted_host2>
+port: <decrypted_port2>
+passowrd: <decrypted_password2>
+~
+===
+```
+
+Note that the agent needs to be restarted to pick up new configuration changes.
```

Comment:
`to pick up new configuration changes` -> `to pick up changes on config files` (just want to mention that it needs to be restarted for files but not for other config sources)